core/src/test/java/kafka/test/annotation/ClusterTestDefaults.java

```diff
@@ -39,4 +39,5 @@
     int brokers() default 1;
     int controllers() default 1;
     boolean autoStart() default true;
+    ClusterConfigProperty[] serverProperties() default {};
```

Comment:
Could you add comments for this new fields?

Reply:
Yes, updated it. Thank you.